src/main/java/com/redhat/foreman/ForemanAPI.java

```diff
@@ -103,7 +103,12 @@ public JsonNode reserveHost(String hostname) {
      * @return string to be used for reserving.
      */
     private String getReserveReason() {
-        return "Reserved for " + Jenkins.getInstance().getRootUrl();
+        String url = "unknown";
+        Jenkins instance = Jenkins.getInstance();
+        if (instance != null) {
+            url = instance.getRootUrl();
```

Comment:
`getInstace()`/`getActiveInstance()` is a mess. None of them can return null on master in common times and findbugs will not help you to catch the cases when it might be null: executing on slave, Jenkins is booting/terminating, etc.

IIRC there is a version of Jenkins core that gets this right-ish: one of them throws or returns value (the one we should use) and the other that requires check. I presume that is newer than the version we depend on. So for the time being, let's use *some* (I do not expect it would matter for this plugin) variant and let findbugs to ignore `Jenkins.get(Active)Instance`.